ec-divisors-contest/benches/divisors.rs

```diff
@@ -22,12 +23,12 @@ fn bench_scalar_mul_divisors(c: &mut Criterion) {
 
     // Run the benchmark for the reference implementation
     group.bench_function("reference-impl", |b| {
-        b.iter(|| run_bench_ref(&point, &rand_scalar))
+        b.iter_with_large_drop(|| run_bench_ref(&point, &rand_scalar))
```

Comment:
`run_bench_ref` and `run_bench_contest` return structs containing `Vec`s which causes the dellocation time to be included in the benchmark. This probably won't matter due to modern allocator speeds although it does remove a variable.

https://bheisler.github.io/criterion.rs/book/user_guide/timing_loops.html#iter_with_large_drop